lib/project_types/rails/commands/generate.rb

```diff
@@ -23,15 +23,26 @@ def self.extended_help
             {{cyan:webhook}}: Generate and register a new webhook that listens for the specified Shopify store event.
               Usage: {{command:#{ShopifyCli::TOOL_NAME} generate webhook [type]}}
           {{bold:Examples:}}
-            {{cyan:shopify generate webhook PRODUCTS_CREATE}}
+            {{cyan:#{ShopifyCli::TOOL_NAME} generate webhook PRODUCTS_CREATE}}
               Generate and register a new webhook that will be called every time a new product is created on your store.
         HELP
       end
 
       def self.run_generate(script, name, ctx)
         stat = ctx.system(script)
         unless stat.success?
-          raise(ShopifyCli::Abort, CLI::UI.fmt(response(stat.exitstatus, name)))
+          ctx.abort(response(stat.exitstatus, name))
```

Comment:
`ctx.abort()` will put the `{{x}} ` before the message anyway, so the `{{x}} ` at the start of each return string from `self.response()` may be generating TWO 'x' characters in output, for example, the output for `code` != 1 or 2 (the catch-all case) will likely be
```
x x Error
```
If this is the case, suggest to remove the `{{x}} ` from the the messages in `self.response()`

Reply:
ooh, good call!